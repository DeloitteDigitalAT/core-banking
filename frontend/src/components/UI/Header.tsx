import React from "react";

const Header: React.FC = properties => {
  return <div className="text-xl py-4 mb-0 text-gray-800">{properties.children}</div>;
};

export default Header;
