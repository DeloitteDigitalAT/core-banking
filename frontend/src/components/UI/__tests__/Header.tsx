import React from "react";
import renderer from "react-test-renderer";
import Header from "../Header";

describe("UI/Header component", () => {
  test("snapshot renders", () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
