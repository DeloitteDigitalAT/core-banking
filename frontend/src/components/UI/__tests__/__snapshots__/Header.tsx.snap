// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI/Header component snapshot renders 1`] = `
<div
  className="text-xl py-4 mb-0 text-gray-800"
/>
`;
